### PR TITLE
Add destination filter

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

**Review by saruniitr — APPROVED** [review]
lgtm

File: api/search/application/backends.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def get_search_query_params(self, request):
         return [] if wildcard_present(updated_params) else updated_params
 
 
-class LiteCustomFilteringFilterBackend(FilteringFilterBackend):
+class WildcardAwareFilteringFilterBackend(FilteringFilterBackend):
     def get_filter_query_params(self, request, view):
         filter_params = super().get_filter_query_params(request, view)
         Wildcard_param_key = f"wildcard{SEPARATOR_LOOKUP_FILTER}wildcard"
```
Comment (r4vi) [inline api/search/application/backends.py:35]: is there a reason for the capital W in `Wildcard` doesn't really match python style guide

File: api/search/application/documents.py [hunk 2]
```diff
@@ -20,74 +12,197 @@
 
 
 part_number_analyzer = analysis.analyzer(
-    "part_number_analyzer",
-    tokenizer=analysis.tokenizer("part_number_path_hierarchy", "path_hierarchy", delimiter="-"),
-    filter=["lowercase", "trim",],
+    'part_number_analyzer',
```
Comment (r4vi) [inline api/search/application/documents.py:15]: I think you should run through `black` as it prefers " over '

**Review by r4vi — COMMENTED** [review]
not sure why there are `lite_content` changes, maybe you need to pull master `lite_content` ?

File: api/search/application/backends.py [hunk 3]
```diff
@@ -20,7 +20,7 @@ def wildcard_present(params, wildcards=["?", "*"]):
     return any([char in p for char in wildcards for p in params])
 
 
-class LiteCustomSearchFilterBackend(SearchFilterBackend):
+class WildcardAwareSearchFilterBackend(SearchFilterBackend):
     def get_search_query_params(self, request):
```
[the remark this answers is not in the document]
Comment (r4vi) [reply api/search/application/backends.py:24]: I don't mind if there's a reason, just wondering

File: api/search/application/backends.py (hunk 1 above)
Comment (saruniitr) [reply api/search/application/backends.py:35]: I added this and didn't notice the captialization, fixing it.

File: api/search/application/views.py [hunk 4]
```diff
@@ -1,52 +1,124 @@
...
     nested_filter_fields = {
-        "clc_rating": {"field": "goods.good.control_list_entries.rating", "path": "goods.good.control_list_entries",},
-        "incorporated": {"field": "goods.incorporated", "path": "goods"},
+        "rating": {"field": "goods.control_list_entries.rating.raw", "path": "goods.control_list_entries",},
+        "destination": {"field": "parties.country.raw", "path": "parties",},
+        "part": {"field": "goods.part_number.raw", "path": "goods",},
+        "incorporated": {"field": "goods.incorporated", "path": "goods",},
     }
 
+    highlight_fields = {"*": {"enabled": True, "options": {"pre_tags": ["<b>"], "post_tags": ["</b>"]}}}
```
Comment (saruniitr) [inline api/search/application/views.py:62]: will it help if we add a background color also? eg light yellow or similar?

File: api/search/application/views.py [hunk 5]
```diff
@@ -1,52 +1,124 @@
...
+
     # Define ordering fields
     ordering_fields = {
         "id": None,
     }
     # Specify default ordering
     ordering = ("id",)
+    def get_queryset(self):
+        import pdb
```
Comment (saruniitr) [inline api/search/application/views.py:71]: leftover debug statements

**Review by saruniitr — APPROVED** [review]
looks good to me, debug statements to be removed

File: api/search/application/views.py (hunk 4 above)
Comment (richtier) [reply api/search/application/views.py:62]: we can let lite-frontend code handle that via css

**Pull request merged**